### PR TITLE
Bump zwave-js to 11.6.0 and zwave-js-server to 1.30.0

### PR DESCRIPTION
**Pull request by raman325**
@AlCalzone anything else you want to highlight in the detailed changelog?

- [Bump Z-Wave JS Server to 1.29.0](https://github.com/zwave-js/zwave-js-server/releases/tag/1.29.0)
- [Bump Z-Wave JS Server to 1.29.1](https://github.com/zwave-js/zwave-js-server/releases/tag/1.29.1)
- [Bump Z-Wave JS Server to 1.30.0](https://github.com/zwave-js/zwave-js-server/releases/tag/1.30.0)
- [Bump Z-Wave JS to 10.23.4](https://github.com/zwave-js/node-zwave-js/releases/tag/v10.23.4)
- [Bump Z-Wave JS to 10.23.5](https://github.com/zwave-js/node-zwave-js/releases/tag/v10.23.5)
- [Bump Z-Wave JS to 10.23.6](https://github.com/zwave-js/node-zwave-js/releases/tag/v10.23.6)
- [Bump Z-Wave JS to 11.0.0](https://github.com/zwave-js/node-zwave-js/releases/tag/v11.0.0)
- [Bump Z-Wave JS to 11.1.0](https://github.com/zwave-js/node-zwave-js/releases/tag/v11.1.0)
- [Bump Z-Wave JS to 11.2.0](https://github.com/zwave-js/node-zwave-js/releases/tag/v11.2.0)
- [Bump Z-Wave JS to 11.3.0](https://github.com/zwave-js/node-zwave-js/releases/tag/v11.3.0)
- [Bump Z-Wave JS to 11.4.0](https://github.com/zwave-js/node-zwave-js/releases/tag/v11.4.0)
- [Bump Z-Wave JS to 11.4.1](https://github.com/zwave-js/node-zwave-js/releases/tag/v11.4.1)
- [Bump Z-Wave JS to 11.4.2](https://github.com/zwave-js/node-zwave-js/releases/tag/v11.4.2)
- [Bump Z-Wave JS to 11.5.0](https://github.com/zwave-js/node-zwave-js/releases/tag/v11.5.0)
- [Bump Z-Wave JS to 11.5.1](https://github.com/zwave-js/node-zwave-js/releases/tag/v11.5.1)
- [Bump Z-Wave JS to 11.5.2](https://github.com/zwave-js/node-zwave-js/releases/tag/v11.5.2)
- [Bump Z-Wave JS to 11.5.3](https://github.com/zwave-js/node-zwave-js/releases/tag/v11.5.3)
- [Bump Z-Wave JS to 11.6.0](https://github.com/zwave-js/node-zwave-js/releases/tag/v11.6.0)

### CODE REVIEW

File: zwave_js/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,70 @@
...
+- Z-Wave JS Server: Add support for lastSeen property
+- Z-Wave JS: Improve support for "identify" commands
+- Z-Wave JS: Add API to discover node neighbors
+- Z-Wave JS: Add support for assigning custom return routes and reading previously set priority return routes from cache
+
+### Bug fixes
+
+- Z-Wave JS Server: Fix hard reset support in Z-Wave JS Server
+- Z-Wave JS: Fix an issue that sometimes occurred when attempting to remove a node from the network
+- Z-Wave JS: Fix handling of incoming S2 multicast commands
```
Comment (AlCalzone) [inline zwave_js/CHANGELOG.md:17]: These are the ones I consider most interesting
```suggestion
### Bug fixes

- Z-Wave JS Server: Fix hard reset support in Z-Wave JS Server
- Z-Wave JS: Increased OTA update timeout, which can help with firmware updates in busy/unstable networks
- Z-Wave JS: Fixed a crash that could happen when including a device with an inclusion controller
- Z-Wave JS: Improved the automatic removal of factory-reset devices that are slow to leave the network
- Z-Wave JS: Devices that failed to join using SmartStart are now automatically removed
- Z-Wave JS: Fix an issue where Z-Wave JS could get stuck when removing a node from the network failed

```

File: zwave_js/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,70 @@
 # Changelog
 
+## 0.1.85
+
+### New features
+
+- Z-Wave JS Server: Add support for inclusions that are started outside of Z-Wave JS Server
```
Comment (AlCalzone) [inline zwave_js/CHANGELOG.md:7]: Does the integration handle this yet?

File: zwave_js/CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,70 @@
 # Changelog
 
+## 0.1.85
+
+### New features
+
+- Z-Wave JS Server: Add support for inclusions that are started outside of Z-Wave JS Server
+- Z-Wave JS Server: Add support for lastSeen property
+- Z-Wave JS: Improve support for "identify" commands
```
Comment (AlCalzone) [inline zwave_js/CHANGELOG.md:9]: Likewise, has support for this landed in HA already?

File: zwave_js/CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,70 @@
...
 
+## 0.1.85
+
+### New features
+
+- Z-Wave JS Server: Add support for inclusions that are started outside of Z-Wave JS Server
+- Z-Wave JS Server: Add support for lastSeen property
+- Z-Wave JS: Improve support for "identify" commands
+- Z-Wave JS: Add API to discover node neighbors
+- Z-Wave JS: Add support for assigning custom return routes and reading previously set priority return routes from cache
```
Comment (AlCalzone) [inline zwave_js/CHANGELOG.md:11]: Assuming the answer to the other questions is yes, I consider those most relevant. The network management / routing stuff is not exposed in HA:
```suggestion
### New features

- Z-Wave JS Server: Add support for inclusions that are started outside of Z-Wave JS Server
- Z-Wave JS Server: Add support for lastSeen property
- Z-Wave JS: Add support for identifying on request of other nodes
- Z-Wave JS: Improved auto-discovery of config parameters
```

**Review by AlCalzone — CHANGES_REQUESTED** [review]
We should also include 11.6.0 (building right now). This way the regressions from 11.2.0 should not affect anyone.

File: zwave_js/CHANGELOG.md (hunk 2 above)
Comment (raman325) [reply zwave_js/CHANGELOG.md:7]: not yet

File: zwave_js/CHANGELOG.md (hunk 3 above)
Comment (raman325) [reply zwave_js/CHANGELOG.md:9]: not yet

**Comment by raman325** [comment]
addon came up clean on my end (tested with a controller without any attached devices)

**Review by frenck — APPROVED** [review]
Looks good to me 👍 

Feel free to merge when ready to ship.

../Frenck

**Pull request merged**